imports/plugins/included/notifications/client/components/notificationDropdown.js

```diff
@@ -93,7 +87,7 @@ class NotificationDropdown extends Component {
           })}
         </ul>
         <div className="dropdown-footer text-center">
-          <Link to={`${this.prefix}/notifications`} data-i18n="notifications.body.viewAll">View All</Link>
```

Comment:
The prefix is needed for when we have multiple shops, I think just removing it and hard-coding the link is going to break the link in some cases. Make sure you have tested this in both multi-shop and single shop scenarios. Ping me if you need help setting that up.